near/omni-bridge/src/lib.rs

```diff
@@ -1098,6 +1098,30 @@ impl Contract {
         )
     }
 
+    #[private]
+    pub fn deploy_token_by_deployer_callback(
```

Comment:
Let's improve the `test_deploy_token_with_huge_metadata_insufficient_deposit` so we have to check that the token can be deployed second time